src/main/java/org/sunbird/portal/department/model/Edata.java

```diff
@@ -0,0 +1,71 @@
+package org.sunbird.portal.department.model;
+
+import java.util.List;
+import javax.annotation.Generated;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({ "state", "props", "login_time" })
+
+public class Edata {
```

Comment:
Need to move this to another new package which should be separate for telemetry related classes.